src/icons/.npmignore

```diff
@@ -0,0 +1 @@
+src
```

Comment:
I wonder if this is truly what we want to do... I don't know if there are any debugging capabilities that not including the raw icons would truly hinder. For now I assume this is fine

Reply:
I think the best reason to exclude this is if we don't want consumers of `design-system-icons` using the raw icons vs using `sprite.svg` -- Excluding it makes it impossible to use the raw icons

(Though there is a bug here --  if we exclude npm install can't build from files that don't exist haha)

Reply:
I guess we're including them then 😂 we can investigate how to lock this (arguably singular) component down more in the future